src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -82,6 +85,21 @@ public static function getSubscribedEvents()
         );
     }
 
+    protected function getExceptionLogLevel(\Exception $exception): string
+    {
+        $logLevel = LogLevel::CRITICAL;
+        if ($exception instanceof HttpExceptionInterface) {
+            $statusCode = $exception->getStatusCode();
+            if (isset($this->httpStatusCodeLogLevel[$statusCode])) {
+                $logLevel = $this->httpStatusCodeLogLevel[$statusCode];
+            } elseif (400 <= $statusCode && $statusCode < 500) {
```

Comment:
This condition looks strange 🤔 if the status code is lower then 500 it's automatically lower then 400.

Reply:
The condition is between 400 and 500... I typically write conditionals like this as a number line (i.e. 400 < x < 500) as it's easier to read.. But seeing you were confused, maybe not for you!
http://llewellynfalco.blogspot.com.au/2016/02/dont-use-greater-than-sign-in.html

Reply:
well.. you are mixing yoda style with non-yoda style. Thats confusing :)

Reply:
fair enough, de-yoda-ed!

Reply:
well.. SF uses yoda style :laughing:

Reply:
Excuse my language (this is just my personal opinion. I respect yours 👍). 

How the hell is `(x < 10 && 5 < x)` more readable then `(x > 5 && x < 10)`. The fact that I need to think about how this construct is executed indicates an issue IMO.
I _like_ Yoda conditions for equals, but overusing them for higher/lower then? 

I guess it makes sense in Perl or something :trollface: (just kidding).
But this feels overdoing something just because we can.

Again. this is my opinion, I respect yours and those of the author 😃

Reply:
I think you're misreading it @sstok
> `(5 < x && x < 10)` which is a stupid option

That is certainly very unreadable.

The suggested ordering is `(5 < x && x < 10)`. This is also the suggested best practice in [Code Complete 2](https://www.amazon.com/Code-Complete-Practical-Handbook-Construction/dp/0735619670) and I _think_ also [Clean Code](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882), although can't recall 100% that one.

But it's your codebase, so I will respect any convention that has been put in place!